packages/api/package.json

```diff
@@ -22,7 +22,7 @@
     "serve-dist-remote": "cumulus-api serve-dist",
     "reset-tables": "LOCALSTACK_HOST=localhost NODE_ENV=test FAKE_AUTH=true cumulus-api reset-tables",
     "link-api-bin": "sh ./bin/link-api-bin.sh",
-    "prepare": "npm run build && npm run link-api-bin && npm run build-lambda-zips",
+    "webpack": "npm run build && npm run link-api-bin && npm run build-lambda-zips",
```

Comment:
Feel free to tell me I'm crazy and webpack is good.

Reply:
No, I think this really should be in a "publish" script, or something like that.

Reply:
What is the motivation for not leaving it as `prepare`?

Reply:
When one of the packages that uses Typescript is installed locally (when we run `lerna bootstrap`), we need to compile those files into `.js` and `.d.ts`. We also need to make sure that compilation is done in the right order, so that a package like `aws-client` is compiled after something it depends on, like `errors`. Using the `prepare` script was the right place to do that. (In addition, the webpages I've read about using Typescript to publish npm packages, [like this one](https://itnext.io/step-by-step-building-and-publishing-an-npm-typescript-package-44fe7164964c), all recommend that `prepare` is the place to compile your Typescript.

Up until now, we've been using the `prepare` script to run webpack in packages that use it. That isn't really something that we want to do in the `prepare` stage, though, but really something that we want to do before publishing or deploying. Moving webpack out of `prepare` makes sense, but I don't think we've found the right place to actually put it yet, though.

Reply:
Fair enough